lib/mayaUsd/utils/util.cpp

```diff
@@ -740,7 +740,7 @@ std::string UsdMayaUtil::prettifyName(const std::string& name)
                 prettyName += ' ';
             }
             prettyName += nextLetter;
-        } else if (name[i] == '_') {
+        } else if (name[i] == '_' || name[i] == ':') {
```

Comment:
Did you run this change by Natalia?

Reply:
Just did. This change is approved. I need to make sure it is called for all UFE attributes, which requires a small change in UsdAttributeHolder.cpp to handle "uiname" metadata for all attributes instead of only NodeGraph and Materials.